pubpub/pubpub
# new: Impact dashboard

Implements embedded impact dashboards from our Metabase install for communities and pubs, with a warning that User data collected before 4/30/2020 used a different system and isn't directly comparable and a coming soon notice for collections (I need to build the collection dash from the ground up because of the way collection data is captured in heap). Dashboards are only visible to users who have `canView` permissions. Users without those permissions, including unauthenticated, are prompted to login or ask their admins for access.

Metabase provides secure embeds via jwt request to its api. To implement this, I added a PubPub API route for requesting the dashboards from Metabase on the server side.

_Test plan_
To test locally, you'll need to update your config.js to include our Metabase api key, now included in 1Pass.

- Visit a community, ie HDSR, while logged into an account with access to it.
- Verify that you can see impact tab at the Community and Pub level.
- Logout and visit an impact tab (or visit from an incognito window). Make sure you see the message prompting to ask the community admin for impact data, and that no requests to the metabase API endpoint are made.
- Make sure you cannot change the Community ID or Pub ID for the dashboards.
